src/test/metrics_test.cpp

```diff
@@ -522,4 +546,54 @@ TEST_F(MetricsTestFixture, MetricsGaugeIgnoresSameSeconds)
     EXPECT_EQ(gauge["time_series"][1][1], 11.0);
 }
 
+TEST_F(MetricsTestFixture, MetricsLiveStatistics)
```

Comment:
Shall the following tests be added:
- bind live + normal metrics, start/stop run, check binding state (is live metric still recording after the run?)
- bind live metric and normal metric, record data, clear history, check reported date on both, validating clear only clears the live data.